CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+
+- All pull request meant to be included in the release should be merged into the `next` branch following standard [review/voting process](/overview/governance#decision-making). It is recommended to include a changelog entry as a part of a pull request. Maintrainers merge these pull requests using "Squash and merge" strategy.
+
+- When the `Release vX.Y` is ready to be released, the maintainer ensure the changelog correctness, resolves conflicts if needed, and merges the `next` branch into the `main` branch using "Create a merge commit" strategy. After the website successfully builds, the maintainer creates a new tag and release on GitHub linking the changelog entry with release notes for the version released.
+
+### Codebase contribution
+
+The Data Package project is based on Astro Starlight static-site generator and uses TypeScript/Node.js for building process. Here are the steps to contribute to the codebase.
+
+1. **Fork the repository**: Click the "Fork" button in the upper right corner of the repository page.
+2. **Clone the repository**: Clone the forked repository to your local machine.
+3. **Install dependencies**: Run `npm install` to install the project dependencies.
+4. **Start a dev server**: Run `npm start` to see the changes locally.
+5. **Make changes**: Make your changes to the codebase.
+6. **Generate profiles**: Run `npm run generate` to generate the profiles.
+7. **Format style**: Run `npm run format` to format the style.
+8. **Run tests**: Run `npm test` to run the tests.
+9. **Commit changes**: Commit your changes to your forked repository.
+10. **Create a pull request**: Create a pull request from your forked repository to the appropriate branch of the main repository (see the Branching Model above).
+
+When a pull request is created, it will be reviewed by the Data Package maintainers. Github Automation creates a live preview site for every new pull request. Once the pull request is approved, it will be merged into the main repository.
+
+Note that the project uses two different types of produced artifacts:
+
+- **npm run generate**: This command generates the profiles from the `profiles` folder and sync profile version in the specs. It is required to run this command after changing the YAML/JSON files in `profiles` folder, and the output has to be committed to the repository.
+
+- **npm run build**: This command builds the project and generates the static site. This command is run by automation and the output is not commited to the repository. As a contributor, you don't need to run this command although you can use `npm run preview` to debug the site built in production-mode locally.
+
+## Backward compatibility
+
+This section outlines the rules for backward compatibility in the Data Package specification.
+
+1. An existing `datapackage.json` that is valid MUST NOT becoming invalid in the future.
```

Comment:
```suggestion
1. An existing `datapackage.json` that is valid MUST NOT become invalid in the future.
```

Reply:
Consider adding an example of how an existing valid `datapackage.json` could become invalid.